src/classes/USER_UserService_TEST.cls

```diff
@@ -287,20 +287,24 @@ public with sharing class USER_UserService_TEST {
     * @description Creates and assigns the permission set with Customize Application flag set
     */
     private static void assignCustomizeApplicationPermissionSet(Id userId) {
+        final String translationPerm = 'PermissionsManageTranslation';
+
         // Create a permission set with CustomizeApplication set ensuring other flags
         // on which CustomizeApplication settings depends are set as well
         PermissionSet permission = new PermissionSet(
             Name = 'PermissionTest',
-            Label = 'PermissionTest'
+            Label = 'PermissionTest',
+            PermissionsManageCustomPermissions = true,
+            PermissionsViewRoles = true,
+            PermissionsViewSetup = true,
+            PermissionsCustomizeApplication = true
         );
 
-        // Dynamically adding permissions to avoid dependencies, previously our package failed to deploy
-        // to an org without translations enabled, due to the field not existing.
-        permission.put('PermissionsManageCustomPermissions',true);
-        permission.put('PermissionsViewRoles',true);
-        permission.put('PermissionsViewSetup',true);
-        permission.put('PermissionsManageTranslation',true);
-        permission.put('PermissionsCustomizeApplication',true);
+        // Dynamically adding translation permission to avoid dependencies. Previously our package failed to deploy
+        // to an org without translations enabled, since the field did not exist.
+        if (PermissionSet.getSObjectType().getDescribe().fields.getMap().containsKey(translationPerm)) {
```

Comment:
When I tried this in the debug console the key it returned was lowercase.

Reply:
@davisagli  A field map is special and is case insensitive. They keyset itself is case sensitive but I switched to containsKey vs checking the keyset.

Reply:
Got it!